lib/hard-module.js

```diff
@@ -140,22 +140,22 @@ HardModule.prototype.build = function build(options, compilation, resolver, fs,
   // Rendered source used in built output.
   this._renderedSource = new HardSource(this.cacheItem);
 
-  var state = {state: {imports: {}}};
-  this.dependencies = deserializeDependencies.dependencies.call(state, this.cacheItem.dependencies, this);
-  this.variables = deserializeDependencies.variables.call(state, this.cacheItem.variables, this);
-  this.warnings = this.cacheItem.warnings.map(deserializeError(ModuleWarning, state), this);
-  this.errors = this.cacheItem.errors.map(deserializeError(ModuleError, state), this);
-  deserializeDependencies.blocks.call(state, this.cacheItem.blocks, this);
```

Comment:
We're removing all usage of `deserializeDependencies` can we also lose the required dep further up on line 6?